### ISSUE TITLE
lalf: command not found

### ISSUE THREAD

**Issue by ghost** [Roromis/Lalf-Forumactif#59]
Bonjour, 

J'ai réussi sans problème l'installation de LALF, et ensuite procédé à la configuration. Lorsque je lance la commande pour l'exportation, on m'indique qu'elle n'existe pas. Ci joint ce que le terminal affiche:

`thiabaud@UbuAT:~$ sudo pip3 install --user -U https://github.com/Roromis/Lalf-Forumactif/archive/master.zip
[sudo] password for thiabaud: 
Downloading/unpacking https://github.com/Roromis/Lalf-Forumactif/archive/master.zip
  Downloading master.zip (65kB): 65kB downloaded
  Running setup.py (path:/tmp/pip-otw527dj-build/setup.py) egg_info for package from https://github.com/Roromis/Lalf-Forumactif/archive/master.zip

Requirement already up-to-date: pyquery in ./.local/lib/python3.4/site-packages (from lalf==3.0a0)
Requirement already up-to-date: requests in ./.local/lib/python3.4/site-packages (from lalf==3.0a0)
Requirement already up-to-date: Pillow in /usr/local/lib/python3.4/dist-packages (from lalf==3.0a0)
Requirement already up-to-date: lxml>=2.1 in ./.local/lib/python3.4/site-packages (from pyquery->lalf==3.0a0)
Requirement already up-to-date: cssselect>0.7.9 in ./.local/lib/python3.4/site-packages (from pyquery->lalf==3.0a0)
Installing collected packages: lalf
  Found existing installation: lalf 3.0a0
    Uninstalling lalf:
      Successfully uninstalled lalf
  Running setup.py install for lalf

```
Installing lalf script to /home/thiabaud/.local/bin
```

Successfully installed lalf
Cleaning up...
thiabaud@UbuAT:~$ cd Lalf/
thiabaud@UbuAT:~/Lalf$ ls
total 8,0K
-rw-r--r-- 1 thiabaud thiabaud 963 sept.  7 15:17 config.cfg
-rw-r--r-- 1 thiabaud thiabaud 963 sept.  7 15:17 config.cfg~
thiabaud@UbuAT:~/Lalf$ lalf
lalf: command not found
thiabaud@UbuAT:~/Lalf$ 
`

J'ai tenté en copiant le fichier lalf de .local/bin dans le dossier avec le fichier de configuration, mais il se refuse alors car il ne détecte pas la version de LALF.
Merci de votre aide :)

**Comment by chezlenny** [comment]
Ca n'a pas de rapport avec ce problème : https://github.com/Roromis/Lalf-Forumactif/issues/54

**Comment by ghost** [comment]
Non, le problème de #54 était une mauvaise orthographe puis une sql pas crée, mais la commande marchait. Ici le script a été installé, mais la commande n'est pas trouvée et lorsque le fichier lalf est lancé directement, le script se stoppe automatiquement car il ne reconnaît pas la version du LALF.

**Comment by ChrysMa** [comment]
Utiliser
$  ~/.local/bin/lalf 
au lieu de
$ lalf
